tests/data/hostgaplugin/vm_settings.json

```diff
@@ -1,4 +1,6 @@
 {
+    "hostGAPluginVersion":"1.0.8.115",
+    "vmSettingsSchemaVersion":"0.0",
```

Comment:
Just to verify, these fields have always been present in real VmSettings objects, right?

Reply:
They were added post 112, not sure when. We default to 0.0.0.0 if not present